base/applications/network/dwnl/lang/ro-RO.rc

```diff
@@ -0,0 +1,32 @@
+/*
+ * PROJECT:     ReactOS File Download Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Romanian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Uz: dwnl URL [DESTINAȚIE]\n"
+    IDS_RESOLVING "Rezolvare %s... "
+    IDS_DONE "terminat.\n"
+    IDS_LENGTH_FULL "Lungime: %I64u [%s]\n"
+    IDS_LENGTH "Lungime: %I64u\n"
+    IDS_FILE_SAVED "\nFișier salvat.\n"
+    IDS_CONNECTING_TO_FULL "Conectare la %s [%s]... "
+    IDS_CONNECTING_TO "Conectare la %s... "
+    IDS_REDIRECTING_TO "Redirecționare la %s... "
+    IDS_SEND_REQUEST "Trimitere cerere... "
+    IDS_BYTES_DOWNLOADED "%I64u octeți descărcate"
+    IDS_BYTES_DOWNLOADED_FULL "%2d%% (%I64u octeți descărcate)"
```

Comment:
Here we have a disagreement in gender. Also, maybe some other term instead of "a descărca" fits better? Don't forget to adapt, please don't do just literal translation.